data/sql/updates/db_world/2020_12_25_00.sql

```diff
@@ -0,0 +1,629 @@
+-- DB update 2020_12_24_01 -> 2020_12_25_00
```

Comment:
should use pending SQL to add new SQL code

Reply:
we don't have to add SQL code here...idk why there are such files there